_posts/en/newsletters/2021-08-04-newsletter.md

```diff
@@ -82,6 +82,6 @@ guidelines for the addition of new tracepoints.
 [C-Lightning 0.10.1]: https://github.com/ElementsProject/lightning/releases/tag/v0.10.1rc1
 [bitcoin core 22.0]: https://bitcoincore.org/bin/bitcoin-core-22.0/ {% comment %}<!-- FIXME: confirm when uploaded -->{% endcomment %}
 [tracing doc]: https://github.com/bitcoin/bitcoin/blob/master/doc/tracing.md
-[contrib tracing doc]: https://github.com/bitcoin/bitcoin/blob/master/contrib/tracing
+[contrib tracing doc]: https://github.com/bitcoin/bitcoin/blob/84ace9aef116a05e034730f2bb2f109d1d77aac7/doc/tracing.md
```

Comment:
in 0ce1b67c0955bc3a7d847ff59f488d78b1bf42c1:

Both links ([tracing doc] and [contrib tracing doc]) now link to the same document. I think this should be

```markdown
[tracing doc]: https://github.com/bitcoin/bitcoin/blob/8f37f5c2a562c38c83fc40234ade9c301fc4e685/doc/tracing.md
[contrib tracing doc]: https://github.com/bitcoin/bitcoin/tree/8f37f5c2a562c38c83fc40234ade9c301fc4e685/contrib/tracing
```

Reply:
Fixed, thanks @0xB10C!

Reply:
Thanks :)